### DISCUSSION TITLE
Project description on project suggest box or chosen projects

### DISCUSSION THREAD

**Discussion by shusterboris** [oracle/opengrok#4227] [Ideas]
There is very interesting feature: add description to project. The description shows in header of found file in search results. It would be very convenient if the description displayed a project's suggest box (project selection dialog) or a tooltip on the button of the selected project

**Comment by vladak** [comment]
I think that's sort of tracked by #600. As a workaround, one could use the [Messages](https://opengrok.docs.apiary.io/reference/0/messages/adds-message-to-a-system) and/or the [path descriptions](https://github.com/oracle/opengrok/wiki/Webapp-configuration#path-descriptions).

**Comment by shusterboris** [comment]
Thank you!
I really use path description, but they are showing only on xref area (list
of files that were found), but not on main panel. I even ready to add it
jsf-file (I hate to change sources... ☺️)... but it seems description is
not property, so I need to find how to do it... Would be appreciate for
hint...
The messages as workaround are good idea, I implemented it, but my users
become afraid, that are seeing orange mark😊. So, again source codes...
But in any case, OpenGrok is very useful tool!

пн, 20 мар. 2023 г., 11:59 Vladimir Kotal ***@***.***>:

**Reply by vladak** [reply]
The messages have various levels (see the Apiary doc) and the color of the exclamation mark icon changes depending on the level. Thanks for the positive feedback !
